backend/src/main/java/gov/cdc/usds/simplereport/service/TestOrderService.java

```
@@ -484,6 +484,16 @@ public void removePatientFromQueue(UUID patientId) {
     _testOrderRepo.save(order);
   }
 
+  @AuthorizationConfiguration.RequirePermissionUpdateTestForPatient
+  public void updateTimerStartedAt(UUID id, String startedAt) {
+    Optional<TestOrder> optionalTestOrder = _testOrderRepo.findById(id);
+    if (optionalTestOrder.isPresent()) {
+      TestOrder order = optionalTestOrder.get();
+      order.setTimerStartedAt(startedAt);
+      _testOrderRepo.save(order);
+    }
+  }
```

should we throw an exception here if we don't find the test order?